### DISCUSSION TITLE
Dogecoin won’t sync, gets stuck at 99% but says the node is synchronised and can’t accept payments. When I reboot it gets stuck on starting. I tried to delete the chain multiple times and it only did the same thing.

### DISCUSSION THREAD

**Discussion by SplatDev** [btcpayserver/btcpayserver#2823] [General]
**Describe the bug**

**To Reproduce the bug**
Steps to reproduce the reported bug:

**Expected behavior**

**Screenshots**

**Your BTCPay Environment (please complete the following information):**
- BTCPay Server Version:    
- Deployment Method:        
- Browser:                  

**Logs (if applicable)**

**Setup Parameters**

**Additional context**

**Comment by NicolasDorier** [comment]
hi, can you provide a screenshot?

**Reply by pavlenex** [reply]
He double posted it here https://github.com/btcpayserver/btcpayserver/issues/2824

**Reply by SplatDev** [reply]
> hi, can you provide a screenshot?

Messed up the original (this post) so I reposted at #2824, thanks!
